include/dfu/dfu_target_suit.h

```diff
@@ -24,28 +24,27 @@ extern "C" {
 /**
  * @brief Set buffer to use for flash write operations.
  *
- * @retval Non-negative value if successful, negative errno otherwise.
+ * @retval Non-negative value if successful, -EINVAL negative errno otherwise.
```

Comment:
IMO `@retval` shall always describe a single return value, so the `-EINVAL` should get a dedicated line -or- the `@return` command should be used.